### PR TITLE
web3torrent: deployment of tracker on heroku via docker

### PR DESCRIPTION
**Pull request by kerzhner**

### CODE REVIEW

File: packages/web3torrent/tracker/package.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "name": "@statechannels/tracker",
```
Comment (snario) [inline packages/web3torrent/tracker/package.json]:
```suggestion
  "name": "@statechannels/web3torrent-tracker",
```

File: packages/web3torrent/tracker/package.json [hunk 2]
```diff
@@ -0,0 +1,8 @@
+{
```
Comment (snario) [inline packages/web3torrent/tracker/package.json]: Maybe add a `script` that calls `packages/web3torrent/tracker/build-push-release.sh`

Comment (kerzhner) [reply packages/web3torrent/tracker/package.json]: 3f8d70bb704af9c9eb26103b209ed484babb1e8b

**Comment by kerzhner** [comment]
@spersico Note that this PR is into your sp/tracker-deploy-tests branch. Feel free to merge it in whenever you see fit.

**Comment by kerzhner** [comment]
fyi @spersico, I removed Procfile as that is not needed with Docker + heroku. f88e9be

**Comment by spersico** [comment]
@kerzhner great, thanks!. Merging now to my PR

**Pull request merged**